dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -553,20 +564,26 @@ private String getJSON(List<Contentlet> cons, HttpServletRequest request, HttpSe
 		return json.toString();
 	}
 
-	public static Set<String> getJSONFields(Structure s) {
+	public static Set<String> getJSONFields(ContentType type) throws DotStateException, DotDataException, DotSecurityException {
 		Set<String> jsonFields=new HashSet<String>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)